src/db/tripledb.plt

```diff
@@ -0,0 +1,23 @@
+:- begin_tests('db/tripledb').
+
+:- use_module(library('db/tripledb'),
+    [ tripledb_load/2,
+      tripledb_ask/3
+    ]).
+
+test('tripledb_load(URL)') :-
+  tripledb:tripledb_load('https://protege.stanford.edu/ontologies/pizza/pizza.owl',
```

Comment:
you do not need to write "tripledb:" as you have imported the predicate in the scope of the file above